src/index.html

```diff
@@ -9,6 +9,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="logo">
+        <a href="#" class="header__link">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="header__link__img"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item ">
+            <a class="nav__link is-active" href="#" >Apple</a>
```

Comment:
```suggestion
            <a class="nav__link is-active" href="#apple">Apple</a>
```
I wrote you about it